### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shrit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <h2 align="center">
   <a href="http://mlpack.org"><img
 src="https://cdn.rawgit.com/mlpack/mlpack.org/e7d36ed8/mlpack-black.svg" style="background-color:rgba(0,0,0,0);" height=230 alt="mlpack: a fast, flexible machine learning library"></a>
```
Comment (conradsnicta) [inline README.md:3]: @shrit the alt string should also be changed accordingly

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
-  <br>a fast, flexible machine learning library<br>
+  <br>a fast, flexible header-only machine learning library<br>
```
Comment (conradsnicta) [inline README.md:4]: @shrit maybe simplify to "header-only machine learning library".

too many qualifiers can muddle the main message

Comment (rcurtin) [reply README.md:4]: Yeah, that seems fine to me, `fast, header-only` gets the idea across still, in my opinion. :+1:

Comment (shrit) [reply README.md:4]: Indeed that can work as well

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <h2 align="center">
   <a href="http://mlpack.org"><img
 src="https://cdn.rawgit.com/mlpack/mlpack.org/e7d36ed8/mlpack-black.svg" style="background-color:rgba(0,0,0,0);" height=230 alt="mlpack: a fast, flexible machine learning library"></a>
-  <br>a fast, flexible machine learning library<br>
+  <br>a fast, header-only machine learning library<br>
```
Comment (zoq) [inline README.md:4]: What is the reason to remove the flexible part? I still think mlpack is flexible.

Comment (shrit) [reply README.md:4]: too many qualifiers

Comment (zoq) [reply README.md:4]: I missed the other comment.

**Review by zoq — APPROVED** [review]
Looks good to me.

**Pull request merged**